### PR TITLE
Global Accessibility Awareness Day

### PR DESCRIPTION
**Pull request by nathanhammond**
Early pass on this. Still actively working on it but hoping people could start looking anyway.

/cc @mixonic

### CODE REVIEW

File: source/blog/2016-05-12-gaad.md [hunk 1]
```diff
@@ -0,0 +1,91 @@
+---
+title: Global Accessibility Awareness Day
+author: Nathan Hammond
+tags: Recent Posts, Ember A11y
+---
+
+May 19, 2016 marks the fifth annual **[Global Accessibility Awareness Day (GAAD)](http://www.globalaccessibilityawarenessday.org/)** and this year we're encouraging you, the Ember.js community, to participate. Not breaking the web has always been a requirement for building ambitious web applications and to that end we must ensure that our framework, addons, and the applications we collectively build maintain are accessible.
```
Comment (mixonic) [inline source/blog/2016-05-12-gaad.md]: > build and maintain

The comparison to accessibility and breaking the web seems a bit thin IMO. Breaking the web is more about standards adherence than it is the experience of a particular user group. Regardless the juxtaposition is perhaps more confusing than helpful?

File: source/blog/2016-05-12-gaad.md [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+- Resource 1
+- Resource 2
+- Resource 3
+- Resource 4
+
+#### Presentations
+
+- [Accessibility Mechanics for Web Applications](https://vimeo.com/163925627) by [Léonie Watson](https://twitter.com/LeonieWatson) given at [Ember London](http://emberlondon.com/).
+- [Building for Accessibility](https://www.youtube.com/watch?v=ok9v9-Tcq0o) by [Nathan Hammond](https://twitter.com/nathanhammond) and [Jennison Asuncion](https://twitter.com/jennison) (one of the co-founders of GAAD) given at [Wicked Good Ember](https://wickedgoodember.com/). [Followup blog post.](http://www.nathanhammond.com/building-for-accessibility)
+- Presentation 3
```
Comment (Robdel12) [inline source/blog/2016-05-12-gaad.md]: If we can't find something better (but please do!) we can add my a11y-debt talk in.

File: source/blog/2016-05-12-gaad.md (hunk 1 above)
Comment (nathanhammond) [inline source/blog/2016-05-12-gaad.md]: I feel like absolute silence after clicking on a link counts as "breaking the web." But I do agree that I could tighten this up. Rewriting a bit.

File: source/blog/2016-05-12-gaad.md (hunk 2 above)
Comment (nathanhammond) [inline source/blog/2016-05-12-gaad.md]: Got you. 😄

**Comment by nathanhammond** [comment]
This is good to go!

**Comment by mixonic** [comment]
@nathanhammond I'll merge, update the date, and deploy in the am. Will ping you when it is live

**Comment by nathanhammond** [comment]
Updated the date, added one new link regarding pairing since you last looked.

**Comment by mixonic** [comment]
@nathanhammond live at http://emberjs.com/blog/2016/05/17/global-accessibility-awareness-day.html, thanks :-D

**Pull request merged**